### ISSUE TITLE
[HL25] Options for pre-update gameplay changes (Grenades, Satchel Charge, Crowbar 1 hit killing Headcrabs)

### ISSUE THREAD

**Issue by marigi64** [ValveSoftware/halflife#3505]
I love the new update but the changes to gameplay such as the grenade throwing or the new Satchel charge controls are really messing up my muscle memory after years and years of playing. 

I play a ton of mods that don't have these changes so an option or console command to revert to the old behaviors would be greatly appreciated.

**Comment by kisak-valve** [comment]
Partial duplicate of #3448.

**Comment by MilkeyWilkey** [comment]
crowbar fix is a godsend, this is the original behavior and I'm glad it's back. grenade throwing is also a feature brought back and I hope it will stay that way. don't mind if it can be turned off but I hope at least these two will stay on by default

**Comment by marigi64** [comment]
Playing through some old builds I found the grenade throw distance was only further in the Day One pre-release Demo and not the retail release (unless there is something I'm missing).

**Comment by MilkeyWilkey** [comment]
it is, but it's much more reliable than accedentally throwing it right in front of yourself while strafing and going backwards

**Comment by marigi64** [comment]
Fair enough, I'm fine with these behaviours being the default but I hope they make them optional since they have been throwing me off.
